web/styles/portico/navbar.css

```diff
@@ -312,6 +312,8 @@ details summary::-webkit-details-marker {
 .top-menu-tab-input:checked ~ .top-menu-submenu {
     opacity: 1;
     visibility: visible;
+    max-height: 80vh;
```

Comment:
Since navbar height is always fixed here, you can use this:

```css
max-height: calc(100vh - var(--navbar-height));
```


Define navbar height at top of the `navbar.css` file:

```css
:root {
    /* Navbar height is important to avoid case studies
       items from overflowing at the bottom. Hence, keep
       it in sync with the current value. */
    --navbar-height: 60px;
}
```